test/runtests.jl

```diff
@@ -92,9 +92,9 @@ let var"'" = Diffractor.PrimeDerivativeBack
     @test @inferred(sin'(1.0)) == cos(1.0)
     @test @inferred(sin''(1.0)) == -sin(1.0)
     @test sin'''(1.0) == -cos(1.0)
-    @test sin''''(1.0) == sin(1.0)  broken = VERSION >= v"1.8"
-    @test sin'''''(1.0) == cos(1.0)  broken = VERSION >= v"1.8"
-    @test sin''''''(1.0) == -sin(1.0)  broken = VERSION >= v"1.8"
+    @test sin''''(1.0) == sin(1.0)  # broken = VERSION >= v"1.8"
```

Comment:
if it's no longer broken, remove the comment.

Reply:
I was figuring it'll be broken again soon enough... but sure:
```suggestion
    @test sin''''(1.0) == sin(1.0)
```